public/modules/custom/grants_metadata/tests/data/liikunta_tapahtuma.data.json

```diff
@@ -109,7 +107,7 @@
             "integrationID": "\/LOCALPAK\/v1\/documents\/cd8fe22b-084b-4543-863b-8fa45a936cc0\/attachments\/10087\/"
```

Comment:
Jos mä ajan näitä testejä omalla koneella, niin osaako tuo ottaa huomioon ton integrationID ympäristöprefiksin?

Reply:
Koska tässä testataan vain ATVSchemaa ja pyöritetään dataa niin tuota arvoa ei käytetä oikeasti mihinkään.